app/src/main/res/xml/settings.xml

```diff
@@ -65,4 +66,9 @@
         android:title="@string/menu_about"
         android:key="@string/pref_key_about" />
 
```

Comment:
I feel like we could add a preference category here for *experimental*, and hide the category if it is under **release** or **beta** build

Reply:
I'd do this later in another commit/PR